next.config.js

```diff
@@ -15,4 +15,5 @@ const nextConfig = {
   },
 };
 
-module.exports = nextConfig;
+(module.exports = nextConfig),
```

Comment:
das verstehe ich nicht, kannst du mir das erklären? 😄

Reply:
das sollte [diese Geschichte hier](https://daily-dev-tips.com/posts/nextjs-add-lang-attribute-to-html-tag/) werden, aber wie gerade schon angesprochen funktioniert es leider nicht so ganz... Frage ist auch im [discussions board](https://github.com/orgs/neuefische/discussions/438)